docs/userstories/v1/egress-control-in-anp/egress-control-cluster-nodes.md

```diff
@@ -0,0 +1,41 @@
+
+### Summary
+
+1. **As a** cluster administrator **I want** to easily block access from
+cluster workloads to k8s-apiserver.
```

Comment:
somewhat nitpicky but this doc is named "egress-control-cluster-nodes", but the apiservers may be running on hosts that aren't nodes

Reply:
ah good point.. let me clarify this to say apiserver on host-networked pods is what the scope means.